skbio/sequence/tests/test_iupac_sequence.py

```diff
@@ -433,6 +433,19 @@ def test_expand_degenerates_with_metadata(self):
                      key=str)
         self.assertEqual(obs, exp)
 
+    def test_to_regex_no_degens(self):
+        seq = ExampleIUPACSequence('ABC')
+        regex = seq.to_regex()
+        self.assertEquals(regex.pattern, str(seq))
```

Comment:
Can you use `assertEqual`? `assertEquals` issues deprecation warnings in Python 3.